services/ymax-planner/src/engine.ts

```diff
@@ -177,7 +178,10 @@ export const makeVstorageEvent = (
 };
 
 type Powers = {
-  evmCtx: Omit<EvmContext, 'signingSmartWalletKit' | 'fetch' | 'cosmosRest'>;
+  evmCtx: Omit<
+    EvmContext,
+    'signingSmartWalletKit' | 'fetch' | 'cosmosRest' | 'kvStore'
+  >;
```

Comment:
Why is `kvStore` extracted from `evmCtx` just to be rejoined in `txPowers`? It doesn't appear to have any other use, and I'm committed to keeping the planner stateless.